src/components/filtered-list/filtered-list-item/filtered-list-item.styles.scss

```diff
@@ -0,0 +1,35 @@
+
+.__filtered-list-item-container {
+  .label {
+    margin-bottom: 4px;
+    margin-left: 4px;
+  }
+}
+
+.__filtered-list-item {
+  background-color: white;
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  display: flex;
+
+
+
+  .content-section {
+    padding: 10px;
+    flex: 1;
+  }
+
+  .row {
+    margin-bottom: 4px;
+  }
+
+  .icon-section {
+    width: 50px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
We could move this property directly to the `.icon-section` selector above, as it should have the same effect.

Reply:
@surdu this is under .icon-section? Do you want me to put it on the whole list instead?

Reply:
not exactly: it's under `.icon-section:hover`. Still works as intended, no worries, it just poped into my eye 😄